commit 5ebdf436dfbed36c0ace433c949ef841d93dfea8 (refs/changes/06/106/2)
Author: Kris Shannon <kris@shannon.id.au>
Date:   2016-07-19T21:23:08+00:00 (3 years, 3 months ago)
    
    joyent/node-triton-tags#1 Port range error message doesn't match the code
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Reviewed by: Cody Mello <cody.mello@joyent.com>
    Approved by: Cody Mello <cody.mello@joyent.com>

diff --git a/lib/index.js b/lib/index.js
index a6f543f..d09c384 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -83,7 +83,7 @@ var validatorFromKey = {
                 if (isNaN(svc.port) || svc.port < 1 || svc.port > 65535) {
                     return format(
                         'invalid "triton.cns.services" tag: service port ' +
-                        'number for %s must be within the range 0 - 65535',
+                        'number for %s must be within the range 1 - 65535',
                         svc.name);
                 }
             }
